docs/configuration-properties.md

```diff
@@ -0,0 +1,26 @@
+Debug vs. Release Configuration Property Documentation
+========================================================
+
+There are a number of properties that are contingent upon the configuration setting chosen. The properties and their corresponding values are documented here to provide transparency and enable users to better create custom configuration modes as well.<br />
```

Comment:
I don't think the `<br />` tag does much, you can just add a normal newline whenever needed instead:

```suggestion
There are a number of properties that are contingent upon the configuration setting chosen. The properties and their corresponding values are documented here to provide transparency and enable users to better create custom configuration modes as well.
```